src/server/saved_objects/client/saved_objects_client.js

```diff
@@ -169,7 +171,8 @@ export class SavedObjectsClient {
         {
           type: object.type,
           updated_at: time,
-          [object.type]: object.attributes
+          [object.type]: object.attributes,
+          ...object.extraBodyProperties
```

Comment:
Used to assign the `spaceId`

Reply:
Let's move this one up "top" as well, like we did for `create`

Reply:
Thanks -- don't know how I missed that...fixed!